bookmarks/tests/test_bookmarks_api.py

```diff
@@ -60,6 +62,22 @@ def test_list_bookmarks_should_filter_by_query(self):
                             expected_status_code=status.HTTP_200_OK)
         self.assertBookmarkListEqual(response.data['results'], [self.bookmark1])
 
+    def test_list_bookmarks_should_filter_by_timestamp(self):
```

Comment:
Tests could be a bit more extensive to cover each param, and verify that each actually returns results - for example the `until_added` case below only checks for an empty result, which might also be a bug. It should be trivial to create a few bookmarks with added/modified dates for the test setup.

Ideally this set of extensive tests would be in `test_queries.py`, which contains tests for all the other query logic. Then have a simpler test case in this file to test integration with the API.